pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
   <groupId>org.folio</groupId>
   <artifactId>edge-inn-reach</artifactId>
-  <version>1.0.6-SNAPSHOT</version>
+  <version>2.0.2-SNAPSHOT</version>
```

Comment:
it should be 2.1.0-SNAPSHOT as the next release from master will minor/major

Reply:
Done